### ISSUE TITLE
.dist-info directory not found

### ISSUE THREAD

**Issue by ColmMcBarron** [ubernostrum/webcolors#6]
This seems to have been introduced when upgrading from 1.7.0 to 1.8.0.

Installing via pip is giving the following error:

`AssertionError: webcolors .dist-info direction not found`

I've locked the version on my end to 1.7.0, so it works.

Here's details about my environment if that helps:

pip: 9.0.1
python: 3.5.1

**Comment by bm371613** [comment]
It can be reproduced on docker image `ubuntu:16.04` with python 3.

**Comment by ubernostrum** [comment]
Hmmm.

**Comment by ubernostrum** [comment]
Most reports of this seem to be from toolchains expecting a wheel and getting a source package. I've just bumped versions and pushed 1.8.1 onto PyPI in both source and wheel forms (previously there was only source). Try that and see if it works?

Also, most of the other instances I can find of this issue in other projects indicate the install succeeds despite the error message.

**Comment by ubernostrum** [comment]
Also also: I'm unable to reproduce using pip 9.0.1 on Python 3.5. But I'll keep poking at it.

**Comment by bm371613** [comment]
`1.8.1` fixes is on `ubuntu:16.04`, thanks!

More details about the problem as I experienced it

```
# pip --version
pip 9.0.1 from /usr/local/lib/python3.5/dist-packages (python 3.5)

# pip install webcolors==1.8.0
Collecting webcolors==1.8.0
  Downloading webcolors-1.8.tar.gz
  Running setup.py (path:/tmp/pip-build-c73bx6t1/webcolors/setup.py) egg_info for package webcolors produced metadata for project name unknown. Fix your #egg=webcolors fragments.
Building wheels for collected packages: unknown, unknown
  Running setup.py bdist_wheel for unknown ... done
  Stored in directory: /root/.cache/pip/wheels/60/b2/01/9e20b456532cba685639e323ea578e519676f55dcbc0f42239
  Running setup.py bdist_wheel for unknown ... error
  Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-c73bx6t1/unknown/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpvworzdbepip-wheel- --py
thon-tag cp35:
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/usr/lib/python3.5/tokenize.py", line 454, in open
      buffer = _builtin_open(filename, 'rb')
  FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-build-c73bx6t1/unknown/setup.py'

  ----------------------------------------
  Failed building wheel for unknown
  Running setup.py clean for unknown
  Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-c73bx6t1/unknown/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" clean --all:
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/usr/lib/python3.5/tokenize.py", line 454, in open
      buffer = _builtin_open(filename, 'rb')
  FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-build-c73bx6t1/unknown/setup.py'

  ----------------------------------------
  Failed cleaning build dir for unknown
Successfully built unknown
Failed to build unknown
Installing collected packages: unknown
Successfully installed unknown-0.0.0

# my_entry_point
Traceback (most recent call last):
  File "/usr/local/bin/my_entry_point", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2927, in <module>
    @_call_aside
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2913, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2940, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 635, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 943, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 829, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'webcolors; extra == "format"' distribution was not found and is required by jsonschema
```

**Comment by ubernostrum** [comment]
Closing this since it now seems to be resolved.
